config-rac-db.yml

```diff
@@ -29,3 +29,10 @@
         loop_var: role_item
       when: create_db|bool
   tags: rac-db-adjustements,rac-db-backups,rac-validation-scripts
+
+- hosts: dbasm
```

Comment:
If `--skip-database-config` is specified to set up a cluster without creating a database, is there a way to still ensure we don't have vulnerable components?   (Or, in other words, is there a somewhere we cloud include `patch-vulns` where it is always executed post-install?

Reply:
LGTM;  `install-sw.yml` definitely sounds like the right place to do this.